php/nano/config.yaml

```diff
@@ -0,0 +1,9 @@
+framework:
+  gitlab: x.laylatichy.x/nano
```

Comment:
`gitlab` keyword is not available. you are the first one entirely on `gitlab`. I've open an issue so as this configuration could be written https://github.com/the-benchmarker/web-frameworks/issues/3226

Reply:
I wanted to change this but I see you created a new branch for that already, cheers for that

didn't check toolsdir earlier for that scenario